### DISCUSSION TITLE
How to merge topology nodes

### DISCUSSION THREAD

**Discussion by wuwen5** [apache/skywalking#7536] [Q&A] [answered]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?

![image](https://user-images.githubusercontent.com/5037807/130408788-5c221d7f-c371-4bd3-a468-6affed0c295f.png)

(1) and (2) are the same dubbo service, (1) detect point is server, (2) detect point is client.

Why are the two nodes not merged? What should I do

**Comment by kezhenxu94** [comment, marked as answer]
(2) is generated by SkyWalking OAP backend when the client side is detected but the server side is not yet detected / registered , so you can just wait some time for the (2) to TTL and be erased by cleaner, and the (2) will be gone.
